### PR TITLE
client.py: add new arguments: "delete" and "update"

### PR DESCRIPTION
**Pull request by yanivbi**
To delete/update the nodes and edges in a given json.
According to issue #376

### CODE REVIEW

**Comment by masco-enovance** [comment]
ok to test

File: contrib/python/api/websocket/client.py [hunk 1]
```diff
@@ -181,14 +193,21 @@ def connect(self):
...
+    parser_update = subparsers.add_parser('update', help='update edges and nodes in the given json files')
+    parser_update.add_argument('file', type=str, help='topology to update')
+
     listen_replay = subparsers.add_parser('listen', help='listen help')
+
     args = parser.parse_args()
 
-    if args.mode == "replay":
-        protocol = WSClientReplayProtocol
+    if args.mode != "listen":
```
Comment (nplanel) [inline contrib/python/api/websocket/client.py]: may write in the positive way 
```
if args.mode == "listen":
   protocol = WSClientDefaultProtocol
else
 ....
```

File: contrib/python/api/websocket/client.py [hunk 2]
```diff
@@ -181,14 +193,21 @@ def connect(self):
...
+        protocol = WSClientModifyProtocol
     else:
         protocol = WSClientDefaultProtocol
 
     client = WSClient("Test", "ws://"+args.analyzer+"/ws",
                       protocol=protocol)
-    client.connect()
+    client.connect()
```
Comment (nplanel) [inline contrib/python/api/websocket/client.py]: add the missing tailing \n

Comment (yanivbi) [inline contrib/python/api/websocket/client.py]: What do you mean? where \n is missing?

**Comment by nplanel** [comment]
LGTM
please squash your commits together

**Comment by masco-enovance** [comment]
run go-fmt

**Comment by eranra** [comment]
Can you change the "replay" mode into "add". Now that we have "delete" and "update", it make sense to call this "add" to align.

**Pull request merged**